merge-headers/index.js

```diff
@@ -10,6 +10,10 @@
  * @return {Headers}
  */
 module.exports = function mergeHeaders(submitted, defaults) {
+    if (typeof Headers !== 'function') {
+        return Object.assign({}, defaults, submitted);
+    }
+
     const headers = new Headers();
```

Comment:
Too risky IMO, you don't know what you're initializing

Reply:
Would you say the same for `new Promise()`?

Reply:
no cause `Promise` is part of node, while Headers is not.
if someone override the `Promise` global is really not this lib concern

Reply:
Same for if someone overrides or polyfills the Headers constructor not according to the specification

Reply:
And also, again. We rely on fetch, which isn't in node. This is a fetch wrapper